core/templates/pages/create-certificate-offering-page/create-certificate-offering-page.component.css

```
@@ -0,0 +1,55 @@
+.oppia-create-certificate-offering {
+  margin: 0 auto 0 auto;
+  max-width: 1080px;
+  padding-bottom: 40px;
+  text-align: left;
+  width: 90%;
+}
+
+.oppia-create-certificate-offering .oppia-create-certificate-offering-title {
+  color: #00609c;
+  font-family: 'Capriola', 'Roboto', Arial, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Capriola" (font-family-name-quotes)

(font-family-name-quotes)